server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -198,6 +210,24 @@ static class RestoreZooCommand {
     String file;
   }
 
+  @Parameters(commandNames = "fate",
+      commandDescription = "Operations performed on the Manager FaTE system.")
+  static class FateOpsCommand {
+    @Parameter(names = {"-c", "--cancel"},
+        description = "<txId>{ <txId>...} Cancel new or submitted FaTE transactions")
+    List<String> txIdList = new ArrayList<>();
+
+    @Parameter(names = "--summary", description = "Print a summary of FaTE transaction information")
+    boolean summarize = false;
+
+    @Parameter(names = {"-j", "--json"}, description = "Print Operations in json")
```

Comment:
Should "Operations" be "transaction". I don't think we call them operations anywhere.

Reply:
I was referring more generically to the overall command. But now I realize that we probably won't print anything else in json exception the transaction so that makes more sense.